### PR TITLE
PIM-5450: Add MongoDb bundle to dev requirements

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                 | A
| ----------------- | ---
| Added Specs       |no
| Added Behats      |no
| Travis CI is ok   |yes
| Changelog updated |no

### CODE REVIEW

**Comment by jmleroux** [comment]
Should we also require `doctrine/mongodb-odm v1.0.0-beta12@dev` ?

File: .travis.yml [hunk 1]
```diff
@@ -2,11 +2,7 @@ language: php
 php:
     - "5.4"
     - "5.5"
-    - "hhvm"
-
-matrix:
-    allow_failures:
-        - php: "hhvm"
+    - "5.6"
```
Comment (jmleroux) [inline .travis.yml]: This CE version should be compatible with 5.6.
HHVM is not supported.

**Comment by jjanvier** [comment]
gtm

**Comment by solivier** [comment]
gtm

**Pull request merged**